ground/src/main/java/com/google/android/ground/system/GoogleApiManager.kt

```diff
@@ -37,29 +38,31 @@ constructor(
    * Displays a dialog to install Google Play Services, if missing. Throws an error if install not
    * possible or cancelled.
    */
-  suspend fun installGooglePlayServices() {
+  suspend fun installGooglePlayServices(): Boolean {
```

Comment:
Adding a `Boolean` return value means the caller needs to address both return value and thrown exceptions as failure states. Since we need to catch exceptions anyway, why not rely on those for error states?